virtual_ecosystem/models/litter/env_factors.py

```diff
@@ -0,0 +1,129 @@
+"""The ``models.litter.env_factors`` module contains functions that are used to
+capture the impact that environmental factors have on litter decay rates. These include
+temperature and soil water potential.
+"""  # noqa: D205
+
+import numpy as np
+from numpy.typing import NDArray
+
+from virtual_ecosystem.models.litter.constants import LitterConsts
+
+
+def calculate_environmental_factors(
+    surface_temp: NDArray[np.float32],
+    topsoil_temp: NDArray[np.float32],
+    water_potential: NDArray[np.float32],
+    constants: LitterConsts,
+) -> dict[str, NDArray[np.float32]]:
```

Comment:
I'm not convinced that nesting these three functions into a single wrapper is worthwhile. The three functions below are all very straightforward and there's essentially no overlap apart from the constants. If you drop this, you have three calls with smaller signatures and then need to add two items to the `decay_rates` signature. It's seems much cleaner.